index.js

```diff
@@ -1,39 +1,115 @@
-'use strict';
-
-const co = require('co');
 const amqplib = require('amqplib');
+const co = require('co');
+
+const DEFAULT_EXCHANGE_TYPE = 'topic';
+const DEFAULT_HEARTBEAT = 10;
```

Comment:
It means that if the client crashes, your queue and all its buffered data will self-destruct in ten seconds...